source/logrepl/snapshot.go

```diff
@@ -29,6 +30,12 @@ import (
 	"github.com/jackc/pgx/v4"
 )
 
+// ErrSnapshotComplete is returned by Next when a snapshot is finished
+var ErrSnapshotComplete = errors.New("ErrSnapshotComplete")
```

Comment:
We could change the error message to contain something more human readable (e.g. "snapshot is completed" or similar). Same for `ErrSnapshotInterrupt`.